docs/source/release_notes.rst

```diff
@@ -3,13 +3,23 @@
 Release Notes
 -------------
 
-Future Release
-==============
+.. Future Release
+   ==============
+       * Enhancements
+       * Fixes
+       * Changes
+       * Documentation Changes
+       * Testing Changes
+   
+       Thanks to the following people for contributing to this release:
+
+v1.4.1 Jan 28, 2022
+===================
     * Changes
         * Temporarily restrict woodwork max version (:pr:`1872`)
```

Comment:
Let's change this to "set upper bound for compatible Woodwork version" since the change will be final for this release

Reply:
fixed